components/AppNavbarProfileSearch.vue

```diff
@@ -75,7 +74,7 @@ watchEffect(() => {
         ?.querySelector('lukso-search') as unknown as HTMLElement
       const inputElement = luksoSearch?.shadowRoot?.querySelector('input')
       inputElement?.focus()
-    }, 10)
+    }, 100)
```

Comment:
maybe create a constant for this type of UI feedback.. normally is INPUT_DELAY, FOCUS_DELAY

Reply:
Thanks for catching, actually this was some of my test that got slipped into PR.